app/renderer/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import "./Slider.scss";
+
+type SliderProps = {
+  max?: string;
+  value?: string;
+  children?: React.ReactNode;
+};
+
+type SliderStates = {
+  value: string;
+  max: string;
+};
+
+export class Slider extends React.Component<SliderProps, SliderStates> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      value: props.value,
+      max: props.max,
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
```

Comment:
I think we have agreed that we will use functional components not classes. If i'm wrong please correct me